net/softethervpn/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2008-2016 OpenWrt.org
```

Comment:
Also I'm not sure what kind of absurd rationale you mean, neither was I accusing you of stealing nor did I doubt the origin of your work - I was merely just asking you to put `# Copyright (C) 2016 Federico Di Marco <fededim@gmail.com>`